autopr/utils/repo.py

```diff
@@ -107,6 +112,8 @@ def repo_to_file_descriptors(repo: Repo, context_window: int, file_chunk_size: i
         if blob.type == 'tree':
             continue
         try:
+            if is_path_ignored(blob.path, ignore_patterns):
+                continue
```

Comment:
Can you move this out of the `try` block?

Reply:
Moved above the `try` block